paasta_tools/kubernetes_tools.py

```diff
@@ -2229,7 +2229,14 @@ def get_all_namespaces(kube_client: KubeClient) -> List[str]:
 
 def ensure_namespace(kube_client: KubeClient, namespace: str) -> None:
     paasta_namespace = V1Namespace(
-        metadata=V1ObjectMeta(name=namespace, labels={"name": namespace})
+        metadata=V1ObjectMeta(
+            name=namespace,
+            labels={
+                "name": namespace,
+                "yelp.com/owner": "compute_infra_platform_experience",
```

Comment:
Currently we use flux to add the `yelp.com/owner=compute_infra_platform_experience` label to the paasta namespace. Since this ensure_namespace creates namespaces for PaaSTA it made sense to still have CIPX as owner of the namespace (even if we don't own the workloads)

Reply:
The operator is a pasta service, setup_k8s_job calls ensure_namespace before deploying anything so the operator namespace will exist

Reply:
correct: but if this is happening from a brand new operator, we'll show up as the namespace owner :)

that said, maybe this isn't that big a deal since they'll probably need to setup a whole host of other things in flux (and can swap the namespace owner label there) - but something to think about :)

Reply:
this is a bit tricky cuz we can get the svc owner but as you said it before reogs happens and ownerships move, how are we gonna keep this namespaces ownership up to date??? Also how about services with multiple owners or no defined ownership???

Reply:
correct :)

the point i was trying to get at was that we (either now or in the future) will need to think about namespace metadata a little more thoroughly to work through all these scenarios and not just hardcode a single team name